manual/manual/cmds/comp.etex

```diff
@@ -132,6 +132,12 @@ The \var{src}".annot" file can be used with the emacs commands given in
 "emacs/caml-types.el" to display types and other annotations
 interactively.
 
+\item["-args" \var{filename}]
+ Read additional newline separated command line arguments from \var{filename}.
```

Comment:
`newline-separated` (hyphenated). Is this sensitive to Unix/Windows line-endings?